com.github.qarmin.czkawka.yaml

```
@@ -0,0 +1,34 @@
+app-id: com.github.qarmin.czkawka
+runtime: org.gnome.Platform
+runtime-version: '3.38'
+sdk: org.gnome.Sdk
+sdk-extensions:
+- org.freedesktop.Sdk.Extension.rust-stable
+command: czkawka_gui
+finish-args:
+- "--share=ipc"
+- "--socket=fallback-x11"
+- "--socket=wayland"
+- "--filesystem=host"
+- "--device=dri"
+build-options:
+  append-path: "/usr/lib/sdk/rust-stable/bin"
+  env:
+    CARGO_HOME: "/run/build/czkawka_gui/cargo"
+modules:
+- name: czkawka_gui
+  buildsystem: simple
+  build-commands:
+  - cargo --offline fetch --manifest-path Cargo.toml
+  - cargo --offline build --release
+  - install -Dm755 ./target/release/czkawka_gui -t /app/bin/
+  - install -Dm644 ./data/icons/com.github.qarmin.czkawka.svg -t /app/share/icons/hicolor/scalable/apps/
+  - install -Dm644 ./pkgs/com.github.qarmin.czkawka.desktop -t /app/share/applications/
+  - install -Dm644 ./data/com.github.qarmin.czkawka.metainfo.xml -t /app/share/metainfo
+  sources:
+  - type: dir
+    path: "."
+  - cargo-sources.json
+  - type: git
+    url: https://github.com/qarmin/czkawka.git
+    tag: 2.3.2
```

```suggestion
  - type: git
    url: https://github.com/qarmin/czkawka.git
    tag: 2.3.2
  - type: dir
    path: "."
  - cargo-sources.json
```

I just tried this, and it worked.

---

Thanks, it fixes issue